packages/adapter-pg/src/pg.ts

```diff
@@ -183,8 +183,12 @@ export class PrismaPgAdapter extends PgQueryable<StdClient> implements SqlDriver
   }
 
   async executeScript(script: string): Promise<void> {
-    // TODO: crude implementation for now, might need to refine it
-    for (const stmt of script.split(';')) {
+    const statements = script
+      .split(new RegExp(/;[\t\r\n ]*/g))
```

Comment:
`new RegExp` looks superfluous, you can just pass the regex literal directly.

A few questions about the regex itself:
1. Why is it important to skip the whitespace? If it is, why don't we skip the whitespace before the first statement?
2. Does the `g` flag really change anything here?